### DISCUSSION TITLE
Assistance Needed with Preparing Lookup Table for LAI and Geology Class in mHM Hydrologic Model Setup

### DISCUSSION THREAD

**Discussion by ankur9217** [mhm-ufz/mHM#98] [Q&A] [answered]
Hello,
I am currently setting up the mHM hydrologic model for my study area, the Rapti watershed. As part of preparing the morphological input data, I am encountering difficulties in creating the lookup tables for LAI (Leaf Area Index) and geology class datasets. 
Details of the Issue:
1- LAI Dataset: I need guidance on how to structure the lookup table for the LAI data.
Specifically, how to map the LAI values to the respective categories used by mHM.

2- Geology Class Dataset: I have a raster file (geology_class.asc) containing numerical values representing different geology classes. I am unsure how to create the corresponding lookup table that mHM requires.

Regards

**Comment by StephanThober** [comment, marked as answer]
Dear Ankur,

for the formatting of the look-up tables, you find examples for LAI here

https://git.ufz.de/mhm/mhm/-/blob/develop/test_domain/input/morph/LAI_classdefinition.txt

and geology class here

https://git.ufz.de/mhm/mhm/-/blob/develop/test_domain/input/morph/geology_classdefinition.txt

I hope this helps.
Stephan

**Reply by pallavhydro** [reply]
Hi @ankur9217 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
